src/static/css/mobile-carousel.less

```diff
@@ -0,0 +1,103 @@
+/* ==========================================================================
+   cfgov-refresh
+   Overrides for slick carousel on /the-bureau/ and /careers/working-at-cfpb/.
+   ========================================================================== */
+
+.js-mobile-carousel {
+
+    margin-bottom: unit(45px / @base-font-size-px, em);
+
+    .respond-to-min(1024px, {
+        margin-top: unit(60px / @base-font-size-px, em);
+    });
+
+    .image-container {
```

Comment:
This isn't very BEM. Instead of nesting these, they should probably have a `__mobile-carousel` modifier.